### DISCUSSION TITLE
OPT-090: Applicability to groups of lots

### DISCUSSION THREAD

**Discussion by duncandewhurst** [OP-TED/eForms-SDK#135] [Q&A] [answered]
(Related to https://github.com/open-contracting/european-union-support/issues/71)

`fields.json` includes both OPT-090-Lot and OPT-090-LotsGroup, which suggests that this field can be used in the context of both lots and groups of lots.

However, in https://docs.ted.europa.eu/eforms/1.1.1/schema/all-in-one.html#frameworkAgreementSection, OPT-090 is only mentioned in the context of lots (not groups of lots):

> For a CAN, when, for a given Lot, a Framework Agreement is involved, only the following information is expected at the level of the Lot definition:
> 
> * The framework buyer categories (BT-111) with its associated technical field (OPT-090)

In `fields.json`, BT-111 only appears with a `-Lots` suffix.

Can OPT-090 appear in the context of groups of lots?

**Comment by YvesJo** [comment, marked as answer]
Hi Duncan,
Indeed, BT-111 is only for lots and OPT-090 is not needed for Groups of lots. OPT-090-LotsGroup will be removed.
KR
